src/include/libminiasync/future.h

```diff
@@ -63,8 +63,17 @@
 
 #include <stddef.h>
 #include <stdint.h>
+
+#if defined(__x86_64__) || defined(__amd64__) || defined(_M_X64)	\
+  || defined(_M_AMD64)
 #include <emmintrin.h>
 
+#define __WAIT() _mm_pause()
```

Comment:
`__FUTURE_WAIT()`  ? just to be extra careful :)

Reply:
Done.